travis/configuration.inc.php

```diff
@@ -7,11 +7,16 @@
 	// Feel free to use, change or ignore.
 	define('SERVER_INSTANCE', 'dev');
 
+	/** Define constants here to control the differences between your development, test and production environments. */
+
 	switch (SERVER_INSTANCE) {
 		case 'dev':
+			define('__DESIGN_MODE__', 1);
```

Comment:
For consistency with `configuration.inc.php` in the project. These two files should be as close as possible to ensure proper testing.